sshcommand

```diff
@@ -58,6 +58,12 @@ case "$1" in
 
     getent passwd "$USER" > /dev/null || false
     USERHOME=$(sh -c "echo ~$USER")
+
+    NEW_KEY=$(grep "NAME=$NAME" "$USERHOME/.ssh/authorized_keys" || true)
```

Comment:
Indeed you can, but you will need `docker-machine` to do it. I haven't documented it up yet as I just put it together last week. :)

- install https://www.docker.com/docker-toolbox
- install https://www.virtualbox.org/wiki/Downloads
- create docker machine: `docker-machine create -d virtualbox sshcommand-dev`
- source docker-machine env vars: `eval $(docker-machine env sshcommand-dev`
- run tests: `make test-in-docker`